src/translations/en.json

```
@@ -7676,7 +7681,34 @@
               "time_zones": {
                 "auto": "Use user settings"
               },
-              "no_background": "No background"
+              "no_background": "No background",
+              "analog_options": {
+                "label": "Analog options",
+                "border": {
+                  "label": "Border",
+                  "description": "Show a border around the clock"
+                },
+                "ticks": {
+                  "label": "Ticks",
+                  "description": "Show ticks (indices) on the outside of the clock",
```

As the first choice is "none" the description should include that by slightly rewording:

```suggestion
                  "description": "Whether to show ticks (indices) on the outside of the clock",
```